ui/src/reducers/resource.js

```diff
@@ -128,8 +131,44 @@ const resourceReducer = actions => (state = defaultState, action) => {
         actionError: action.error,
       };
     default:
-      return state;
+      return createClientAppDialog(state, action);
   }
 };
 
+
+function createClientAppDialog(state, action) {
+  switch (action.type) {
+    case 'platform/REGISTER':
+      var newState = {...state};
+      newState.createClientAppDialog.platforms[action.platform.name] = { selected: false};
```

Comment:
This would change the previous state. It should look something like this:
```
var newState = {
  ...state,
  createClientAppDialog: {
    ...state.createClientAppDialog,
    platforms: {
      ...state.createClientAppDialog.platforms,
      [action.platform.name] = { selected: false }
    }
  }
};
```